vendor.conf

```diff
@@ -1,23 +1,25 @@
 github.com/urfave/cli v1.17.0
-github.com/containers/image master
+github.com/containers/image issue-327 https://github.com/hferentschik/image.git
 github.com/opencontainers/go-digest master
 gopkg.in/cheggaaa/pb.v1 ad4efe000aa550bb54918c06ebbadc0ff17687b9 https://github.com/cheggaaa/pb
 github.com/containers/storage master
 github.com/sirupsen/logrus v1.0.0
+github.com/Sirupsen/logrus v1.0.0
```

Comment:
mmm which of these is correct? why do we have both?

Reply:
github.com/sirupsen/logrus v1.0.0 is the correct one now. There was a major issue around logrus when it got renamed from github.com/Sirupsen/logrus to github.com/sirupsen/logrus.

Reply:
right, my question was more "should we keep both?!"